src/Commands/core/UpdateDBCommands.php

```diff
@@ -22,6 +22,12 @@ class UpdateDBCommands extends DrushCommands implements SiteAliasManagerAwareInt
 
     protected $maintenanceModeOriginalState;
 
+    /**
+     * The list of extensions that are installed before updates are applied.
+     * @var array
+     */
+    protected $originalExtensionList;
```

Comment:
Let's remove this section, we no longer need to track if extensions are updated.